test/uk/gov/hmrc/vatsubscription/models/VatCustomerInformationSpec.scala

```diff
@@ -17,11 +17,67 @@
 package uk.gov.hmrc.vatsubscription.models
 
 import assets.TestUtil
+import play.api.libs.json.Json
+import uk.gov.hmrc.vatsubscription.helpers.BankDetailsTestConstants.bankDetailsModelMax
 import uk.gov.hmrc.vatsubscription.helpers.CustomerInformationTestConstants._
+import uk.gov.hmrc.vatsubscription.helpers.PPOBTestConstants.ppobModelMax
 
 
 class VatCustomerInformationSpec extends TestUtil {
 
+  ".pendingBankDetails method" when {
+
+    "there are pending bank detail changes" should {
+
+      "return the pending changes" in {
+        customerInformationModelMax.pendingBankDetails shouldBe Some(bankDetailsModelMax)
+      }
+    }
+
+    "there are NO pending bank detail changes" should {
+
+      "return None" in {
+        customerInformationModelMin.pendingBankDetails shouldBe None
+      }
+    }
+  }
+
+  ".pendingPPOBAddress method" when {
+
+    "there is a pending PPOB Address change" should {
+
+      "return the pending changes" in {
+        customerInformationModelMax.pendingPPOBAddress shouldBe Some(ppobModelMax.address)
+      }
+    }
+
+    "there is NO pending PPOB Address change" should {
+
+      "return None" in {
+        customerInformationModelMin.pendingPPOBAddress shouldBe None
+      }
+    }
+  }
+
+
+  ".pendingEmailAddress method" when {
+
+    "there is a pending Email Address change" should {
+
+      "return the pending changes" in {
+        customerInformationModelMax.pendingContactEmail shouldBe ppobModelMax.contactDetails.flatMap(_.emailAddress)
```

Comment:
I think this would be better as either: 
`shouldBe contactDetailsModelMax.emailAddress`
or
`shouldBe Some(email)`